config.dsc

```diff
@@ -136,6 +136,8 @@ config({
                 // RocksDb
                 { id: "RocksDbSharp", version: "5.8.0-b20181023.3", alias: "RocksDbSharpSigned" },
                 { id: "RocksDbNative", version: "5.14.3-b20181023.3" },
+                
+                { id: "JsonDiffPatch.Net", version: "2.1.0" },

```

Comment:
>{ id: "JsonDiffPatch.Net", version: "2.1.0" }, [](start = 16, length = 46)

I have registered the OSS usage of this package version on the Microsoft OSS website.